odxtools/dynamiclengthfield.py

```diff
@@ -55,7 +55,7 @@ def encode_into_pdu(self, physical_value: ParameterValue, encode_state: EncodeSt
         odxassert(encode_state.cursor_bit_position == 0,
                   "No bit position can be specified for dynamic length fields!")
 
-        if not isinstance(physical_value, list):
+        if not isinstance(physical_value, (tuple, list)):
```

Comment:
why not take any collection?

Reply:
good point. fixed: https://github.com/mercedes-benz/odxtools/pull/300/commits/7c5a56c68d505a33554d8006afaaec370c3c95b6 (I think that `typing.Sequence` is the thing that we want here, though...)